docs/install/windows.md

```diff
@@ -1,37 +1,392 @@
-*  `pwsh` (grab the latest PowerShell release on GitHub: [https://github.com/PowerShell/PowerShell/releases/latest](https://github.com/PowerShell/PowerShell/releases/latest))
+### Step 1: (If installing neuron on a Windows Virtual Machine) 
+
+Create a new VM 
+  
+------------------------------------------------------------ 
+
+### Step 2: Install Git 
+
+Git: [https://git-scm.com/](https://git-scm.com/)
+
+Use this: [https://git-scm.com/download/win](https://git-scm.com/download/win)
+
+Note: Do not use: `winget install --id Git.Git -e --source winget` 
+ 
+  
+------------------------------------------------------------ 
+
+### Step 3: Install Chocolatey [https://chocolatey.org/](https://chocolatey.org/) by command line: 
+
+```bash
+Set-ExecutionPolicy Bypass -Scope Process -Force; [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072; iex ((New-Object System.Net.WebClient).DownloadString('https://community.chocolatey.org/install.ps1')) 
+```
+
+Ensuring Chocolatey commands are on the path 
+
+Ensuring chocolatey.nupkg is in the lib folder 
+ 
+ 
+------------------------------------------------------------ 
+
+### Step 4: Install Visual Studio with C++ 
+
+Use this: [https://visualstudio.microsoft.com/vs/features/cplusplus/](https://visualstudio.microsoft.com/vs/features/cplusplus/) 
+
+ 
+------------------------------------------------------------ 
+
+### Step 5: Get PowerShell 7 or newer version
+
+We will use `pwsh` command - correspond to PowerShell 7. The PowerShell 5 use `powershell` command.  
+ 
+#### Step 5.A: See PowerShell Version Installed: 
+ 
+`$PSVersionTable` or `get-host|Select-Object`  
+ 
+#### Step 5.B: See available options: 
+
+`winget search Microsoft.PowerShell` 
+ 
+#### Step 5.C: Install New PowerShell 
+
+`winget install --id Microsoft.Powershell --source winget`  
+or
+`winget install --id Microsoft.Powershell.Preview --source winget` 
+ 
+#### Step 5.D: Ensure you are working with PowerShell 7
+Note: Find path file to version 7 (`C:\Program Files\PowerShell\7` or `C:\Program Files\PowerShell\7-preview`) and create a shortcut for this powershell 7
+Windows will continue to use 5 version if we try open "PowerShell" 
+ 
+------------------------------------------------------------ 
 
 ## Build Environment installation
 
-You can follow the lines of the scripts from the `ci` folder. The scripts can also be launched as-is for local installation. Perform a git clone of the `nrn` repository.
+### Step 6: Git clone
+At PowerShell 7, go to 
+`C:\Users\User>` and create folder named “Neuron”: 
 
-From a powershell with **administrator** priviledges, run:
-```powershell
-PS C:\workspace\nrn> .\ci\win_download_deps.cmd
 ```
-This will download all required installers and corresponding versions(`Python`, `MPI`, `nsis`)
+mkdir Neuron
```

Comment:
personal opinion: since the clone is going to create a new folder anyways for `nrn`, I'd avoid creating this extra one.

Reply:
That said, since you have instructions that work, maybe it's best to leave it this way because I could imagine accidentally creating a mistake if we tried to get rid of the extra folder now.

Reply:
We can not. We need a Folder (here named as Neuron) in order to maintain a clary separation between the NEURON installed (the C:\nrn\ or C:\Users\User\nrn\) and the NEURON code (C:\Users\User\Neuron\nrn\)